llvm/lib/CodeGen/CMakeLists.txt

```diff
@@ -250,6 +250,7 @@ add_llvm_component_library(LLVMCodeGen
   VirtRegMap.cpp
   WasmEHPrepare.cpp
   WindowScheduler.cpp
+  WindowsHotPatch.cpp
```

Comment:
Similarily, `WindowsSecureHotPatch.cpp`?

Reply:
Renamed to `WindowsSecureHotPatching.cpp`